.github/workflows/post_release_version_bump.yml

```
@@ -0,0 +1,30 @@
+name: Automated Release Version Bumps
+
+on:
+  release:
+    types: [published]
```

Plugin hasn't been post-release version bumped, using action to generate PR 
```suggestion
on:
  push:
  #release:
  #  types: [published]
```